### DISCUSSION TITLE
Unique Identifier used  in deces API

### DISCUSSION THREAD

**Discussion by mzurs** [upptime/upptime#834] [Q&A]
I want to ask about the API that uses a unique identifier to fetch a record **https://deces.matchid.io/deces/api/v1/id/**, so this "id" field is attached to the user after death or before?

**Comment by AnandChowdhary** [comment]
I think this is the wrong place to ask this question, but please correct me if I'm wrong!

**Reply by mzurs** [reply]
Yes, I put it in the wrong place, you can delete this.
